### PR TITLE
Add luci-app-olsr2 application

### PR DESCRIPTION
**Pull request by stargieg**
Signed-off-by: Patrick Grimm <patrick@lunatiki.de>

### CODE REVIEW

**Comment by aparcar** [comment]
Could you post some screenshots? I'm just curious...

**Comment by stargieg** [comment]
[luci-olsr2.pdf](https://github.com/openwrt/luci/files/3000472/luci-olsr2.pdf)

**Comment by aparcar** [comment]
looks cool, I'd merge if you have nothing to change?

File: modules/luci-base/luasrc/cbi/datatypes.lua [hunk 1]
```diff
@@ -49,6 +49,9 @@ end
 function neg(v, ...)
 	return _M['or'](v:gsub("^%s*!%s*", ""), ...)
 end
+function negm(v, ...)
```
Comment (aparcar) [inline modules/luci-base/luasrc/cbi/datatypes.lua]: could you explain this data type?

**Comment by feckert** [comment]
@stargieg ping

**Comment by feckert** [comment]
@aparcar no reaction from pr creater.
should we close this?

**Comment by feckert** [comment]
Nothing happend! I will close this for now.
Please feel free to reopen the pullrequest again if you rebased and fixed the changes.

**Comment by aparcar** [comment]
Sorry for the late response, I guess keep it closed until he responds!

**Pull request closed without merging**